serverless/service_mesh/serverless-ossm-custom-domains.adoc

```diff
@@ -26,7 +26,8 @@ You can customize the domain for your Knative service by configuring the service
 
 * Install the xref:../../serverless/admin_guide/installing-openshift-serverless.adoc#installing-openshift-serverless[{ServerlessOperatorName}] and xref:../../serverless/admin_guide/installing-knative-serving.adoc#installing-knative-serving[Knative Serving].
 * Install xref:../../service_mesh/v1x/preparing-ossm-installation.adoc#preparing-ossm-installation-v1x[{ProductName}].
-* Complete the configuration steps in xref:../../serverless/networking/serverless-ossm.adoc#serverless-ossm[Using {ProductShortName} with {ServerlessProductName}].
+// v2x not supported? Ask @Kenjiro
```

Comment:
@nak3 question: is the procedure for custom domains using service mesh v2.x different than that for v1x?

Reply:
No, it is same procedure between service mesh v1 and v2.

Reply:
Thanks Kenjiro! 🙂